modules/aws/mx/README.md

```diff
@@ -0,0 +1,91 @@
+# DSF DAM MX
+[![GitHub tag](https://img.shields.io/github/v/tag/imperva/dsfkit.svg)](https://github.com/imperva/dsfkit/tags)
+
+This Terraform module provisions a DSF DAM Management server (AKA mx) on AWS as an EC2 instance.
+
+Imperva’s Management Server Manager is a multi-domain management solution that dramatically improves the operational efficiency of managing SecureSphere deployments with multiple MX Management Servers. Management Server Manager meets the operational scalability demands of large enterprises and Managed Security Service Providers by consolidating the management, visibility and reporting across multiple SecureSphere MX Management Servers, and provides system wide health metrics and statistics. 
```

Comment:
This is a description of SOM, not MX.
I think it's better not to use a new term - "SecureSphere", we have enough terms, this is the first time we use it.